_posts/current/blog/2020-05-07-linaro-engineering-highlights-april-2020-new-version-2.md

```diff
@@ -0,0 +1,161 @@
+| Eric Biggers        | 122 | 1.0% | Greg Kroah-Hartman  | 11931 | 1.8% |
+| Arnd Bergmann       | 114 | 0.9% | Chris Wilson        | 10615 | 1.6% |
+| Zheng Bin           | 110 | 0.9% | Srinivas Kandagatla | 8739  | 1.3% |
+| Geert Uytterhoeven  | 103 | 0.9% | Alex Maftei         | 8581  | 1.3% |
+| Greg Kroah-Hartman  | 103 | 0.8% | Maxime Ripard       | 7521  | 1.1% |
+| Masahiro Yamada     | 94  | 0.7% | Peter Ujfalusi      | 6970  | 1.0% |
+| Colin Ian King      | 92  | 0.7% | Tony Lindgren       | 6320  | 0.9% |
+| Ben Skeggs          | 91  | 0.7% | Helen Koike         | 5789  | 0.9% |
+| Ville Syrjälä       | 90  | 0.7% | Takashi Iwai        | 5622  | 0.8% |
+| Andy Shevchenko     | 88  | 0.7% | Shuming Fan         | 5604  | 0.8% |
+| Russel King         | 88  | 0.7% | Michal Kalderon     | 5445  | 0.8% |
+| Alex Deucher        | 86  | 0.7% | Sricharan R         | 5065  | 0.7% |
+| Krzysztof Kozlowski | 82  | 0.6% | Andrii Nakryiko     | 4857  | 0.7% |
+| Thomas Zimmermann   | 80  | 0.6% | Roman Li            | 4852  | 0.7% |
+| Jens Axboe          | 77  | 0.6% | Thierry Reding      | 4845  | 0.7% |
+| Jani Nikula         | 74  | 0.6% | Sunil Goutham       | 4762  | 0.7% |
+
+Congratulations to Arnd, Srinivas, Linua and Naresh for being top contributors to the 5.6 kernel and a thank you to all those who keep Linaro in the top ten Linux Kernel contributors every release.
+
+### LTP (Linux Test Project)
+
+{% include image.html path="/assets/images/content/CORE ENG.svg" class="medium-inline left" alt="CORE ENG.svg" %}
+
+Linaro had been asked by the Members to work on the Linux Test Project (aka LTP) and enhance it to cover all the syscalls in the Linux Kernel. With best effort staffing, work had been progressing slowly. In January, Viresh Kumar (KWG) was able to take up this work and put in a sustained effort, quickly closing the gap and adding support for the following syscalls:
+
+* pidfd_open Io_pgetevents
+* Fsopen
+* Fsconfig
+* Fsmount
+* Fspick
+* Open_tree
+* Move_mount
+* Clone3
+* Openat2
+
+All of the above have been merged,  while work is in progress for three syscalls related to io_uring supported by an ARM member engineer. In addition, twenty new syscalls have been added to the task, all related to the time64 variants and these are now underway. Further information is available in [KWG-326.](https://projects.linaro.org/browse/LKQ-43)
+
+### [PSA Level 1 Certification Showcase](https://www.psacertified.org/app/uploads/2020/04/PSA_Certified-Zephyr_Partner_Showcase.pdf)
+
+Linaro IoT and Embedded (LITE)
+
+Kevin Townsend, LITE Senior Engineer, completed certification of the TF-M integration with Zephyr. This effort was featured on the PSA Certified website in the Partner [showcase](https://www.psacertified.org/partner-showcase/).
+
+![](/assets/images/content/LITE col.svg)
+
+{% include image.html path="/assets/images/content/LITE col.svg" class="small-inline left" alt="LITE col.svg" %}
```

Comment:
Can you replace this with a valid png image? Also you've duplicated your images here. One in markdown and one using the image.html include.